python/ray/ray_constants.py

```diff
@@ -72,7 +72,7 @@ def env_bool(key, default):
 PLASMA_RESERVABLE_MEMORY_FRACTION = 0.69
 
 
-def round_to_memory_units(memory_bytes, round_up):
+def round_bytes(memory_bytes, round_up):
```

Comment:
The previous name is more accurate

Reply:
+1

Reply:
But the resulting units are still in bytes?